apps/web/pages/trials/edit/[id].tsx

```diff
@@ -56,7 +64,16 @@ const EditorEditTrialPage = () => {
     if (editTrialResponse) {
       setLastSaved(editTrialResponse.updatedAt);
       const trial = structuredClone(editTrialResponse)
-      const ctml_json = trial.ctml_jsons[0].data;
+      const ctml_json = trial.ctml_jsons[0].data; 
+    if(ctml_json.prior_treatment_requirements && ctml_json.prior_treatment_requirements.length>0)
+    {
+      const transformedPriorTreatmentRequirements = ctml_json.prior_treatment_requirements && ctml_json.prior_treatment_requirements?transformPriorTreatmentRequirements(ctml_json.prior_treatment_requirements): {};
```

Comment:
confusing code, you are in the if ctml_json.prior_treatment_requirements is not null block already, but you are doing another check with the &&.

Reply:
I am doing two checks here because I making sure to add the below changes to the schema only if the data in prior_treatment_requirements is in array format. So, lets say if the data is in object format, the below changes doesn't apply.

Reply:
In the if condition:
1. ctml_json.prior_treatment_requirements checks if it's defined
2. ctml_json.prior_treatment_requirements.length>0 will throw exception if it's an object ie. {}

Once inside the If block
1. ctml_json.prior_treatment_requirements is true already by 1st check in the if condition, so no need to check again
2. In line 63, you are doing if (ctml_json.prior_treatment_requirements && ctml_json.prior_treatment_requirements) then transformPriorTreatmentRequirements(...) else {}, the if condition will always be true because we are in the if block